### ISSUE TITLE
Crash with error IllegalStateException: No block found for eucalyptus2 PLANKS non-fireproof

### ISSUE THREAD

**Issue by viveleroi** [LatvianModder/YABBA#40]
Using YABBA-1.12-1.0.0, forestry 1.12.1-5.5.4.187, first of all, forge claims the yabba:barrel item is now missing, worrying me that it'll remove all barrels I had in my save:

```
There are 3 missing entries in this save.
If you continue the missing entries will get removed.
A world backup will be automatically created in your saves directory.

Missing minecraft:blocks:
    yabba:barrel
Missing minecraft:items:
    yabba:upgrade
    yabba:barrel
```

But more critically, YABBA crashes with this error:

```
java.lang.IllegalStateException: No block found for eucalyptus2 PLANKS non-fireproof
	at forestry.arboriculture.WoodAccess.getBlock(WoodAccess.java:239)
	at com.latmod.yabba.integration.ForestryIntegration.registerSkins(ForestryIntegration.java:29)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_565_ForestryIntegration_registerSkins_YabbaSkinsEvent.invoke(.dynamic)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:90)
	at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:179)
	at com.feed_the_beast.ftbl.lib.EventBase.post(EventBase.java:18)
	at com.latmod.yabba.client.YabbaClient.loadModelsAndSkins(YabbaClient.java:169)
	at com.latmod.yabba.client.BarrelModelLoader.func_110549_a(BarrelModelLoader.java:60)
	at net.minecraft.client.resources.SimpleReloadableResourceManager.func_110542_a(SimpleReloadableResourceManager.java:121)
	at net.minecraftforge.client.model.ModelLoaderRegistry.registerLoader(ModelLoaderRegistry.java:74)
	at com.latmod.yabba.YabbaItems.registerModels(YabbaItems.java:95)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_310_YabbaItems_registerModels_ModelRegistryEvent.invoke(.dynamic)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:90)
	at net.minecraftforge.fml.common.eventhandler.EventBus$1.invoke(EventBus.java:143)
	at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:179)
	at net.minecraftforge.fml.client.FMLClientHandler.fireSidedRegistryEvents(FMLClientHandler.java:1091)
	at net.minecraftforge.fml.common.FMLCommonHandler.fireSidedRegistryEvents(FMLCommonHandler.java:757)
	at net.minecraftforge.fml.common.Loader.preinitializeMods(Loader.java:606)
	at net.minecraftforge.fml.client.FMLClientHandler.beginMinecraftLoading(FMLClientHandler.java:266)
	at net.minecraft.client.Minecraft.func_71384_a(Minecraft.java:466)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:377)
	at net.minecraft.client.main.Main.main(SourceFile:123)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
```

**Comment by LatvianModder** [comment]
`yabba:barrel item is now missing, worrying me that it'll remove all barrels I had in my save`
Yes that is true, all old barrels will be gone. thus major version change, 1.0.0

**Comment by xKnightGamingx** [comment]
I hope this will be fixed asap cause I need it fixed for my sky pack

**Comment by viveleroi** [comment]
@xKnightGamingx It's a breaking change which is why they bumped the major version number. It's intentional, not something to be fixed.

Eventually I'll take stuff out my barrels and then update.

**Comment by EmosewaGamer** [comment]
I just hope that the crash is fixed. I have many recipes in my pack that use the infinite storage upgrade, so I cannot update my pack until this is fixed. :|

**Comment by LatvianModder** [comment]
Fixed in 4.1.1
